zebedee-cms/pom.xml

```diff
@@ -173,7 +174,8 @@
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-web</artifactId>
-            <version>4.3.8.RELEASE</version>
+            <version>${spring.version}</version>
+            <scope>test</scope>
```

Comment:
I think this dependency can actually be removed now. I had a look and it appears this was the only place using it.

Reply:
Thanks for fixing the test. This is now merged into this branch. I approve your changes ✅.